cmd/format-xl.go

```diff
@@ -632,8 +632,8 @@ func initFormatXLMetaVolume(storageDisks []StorageAPI, formats []*formatXLV3) er
 	for index := range storageDisks {
 		index := index
 		g.Go(func() error {
-			if formats[index] == nil || storageDisks[index] == nil {
-				// Ignore create meta volume on disks which are not found.
+			if formats[index] == nil || storageDisks[index] == nil || !storageDisks[index].IsLocal() {
+				// Ignore create meta volume on disks which are not found or not local.
```

Comment:
Since most disks are not going to be local, it seems a majority of these go routines are going to return while doing no work. It's more efficient to only start go routines for local disks.

Reply:
@vadmeste ^^

Reply:
> Since most disks are not going to be local, it seems a majority of these go routines are going to return while doing no work. It's more efficient to only start go routines for local disks.

Fixed !